src/examples/y_demo_failures.clas.testclasses.abap

```diff
@@ -1,56 +1,19 @@
 *"* use this source file for your ABAP unit test classes
-CLASS ltc_error_fail DEFINITION FOR TESTING
-  RISK LEVEL HARMLESS
-  DURATION SHORT.
-
+CLASS local_test_class DEFINITION FOR TESTING RISK LEVEL HARMLESS DURATION SHORT.
   PRIVATE SECTION.
-    DATA cut TYPE REF TO y_demo_failures.
-
-    METHODS:
-      setup,
-      db_access_in_ut      FOR TESTING,
-      external_call_in_ut  FOR TESTING.
+    METHODS db_access_in_ut FOR TESTING.
+    METHODS external_call_in_ut FOR TESTING.
 ENDCLASS.
 
-CLASS ltc_error_fail IMPLEMENTATION.
-  METHOD setup.
-    cut = NEW y_demo_failures( ).
-  ENDMETHOD.
+CLASS local_test_class IMPLEMENTATION.
 
   METHOD db_access_in_ut.
-    cut->attribute_1 = 'DSAG'.
-    SELECT * FROM t100 UP TO 1 ROWS INTO @DATA(result). "#EC DB_ACCESS_UT
-    ENDSELECT.
-    cl_abap_unit_assert=>assert_equals(
-      act = 1
-      exp = 1 ).
+    SELECT SINGLE * FROM t100
+    INTO @DATA(entry).
   ENDMETHOD.
 
   METHOD external_call_in_ut.
-    DATA dest TYPE RFCDEST VALUE 'NONE'.
-    DATA date TYPE sy-datum.
-    DATA lo_gui_alv_cont TYPE REF TO cl_gui_alv_grid.
-    DATA(bool) = cl_gui_alv_grid=>offline( ).
-
-    SUBMIT demo_program_submit_rep AND RETURN.    "#EC EXT_CALL_UT
-    SUBMIT demo_program_submit_rep AND RETURN.
-
-    CALL FUNCTION 'DATE_TO_DAY' DESTINATION dest
-      EXPORTING
-        date    = sy-datum
-      IMPORTING
-        weekday = date.
-
-    CALL FUNCTION 'DATE_TO_DAY' IN UPDATE TASK
-      EXPORTING
-        date    = sy-datum
-      IMPORTING
-        weekday = date.
-
-    CALL FUNCTION 'DATE_TO_DAY' STARTING NEW TASK 'task' DESTINATION dest
-      EXPORTING
-        date = sy-datum.
-
```

Comment:
I have simplified the external_call_in_ut to report it only once in the example class as the check behavior is being tested on its unit test.

Reply:
By the way, this UT was failing.